### PR TITLE
Containerization

### PR DESCRIPTION
**Pull request by greenw0lf**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+RUN mkdir /root/.DANE /mnt/dane-fs /src /data
+
+WORKDIR /src
+
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+COPY pyproject.toml poetry.lock ./
```
Comment (Veldhoen) [inline Dockerfile:16]: Just a minor improvement would be to move this copy statement after installing poetry (also in the example worker, in fact)

File: docker-entrypoint.sh
Comment (Veldhoen) [inline docker-entrypoint.sh:1]: I got the 'permission denied' error on the entrypoint again, `chmod +x docker-entrypoint.sh` solved this. I'd like to check whether it is also solved for me when you chmod and then push that version to GH.

**Review by Veldhoen — APPROVED** [review]
Yes, looks good and I can build the image and run a container. 
I added two small remarks; I'm mostly curious to see whether you changing the file permissions on entrypoint.sh would also solve the error on my side.

File: docker-entrypoint.sh
Comment (greenw0lf) [reply docker-entrypoint.sh:1]: Well, it doesn't pick up the permission change by default, so I did `git update-index --chmod=+x docker-entrypoint.sh` to update the permissions.

Comment (greenw0lf) [reply docker-entrypoint.sh:1]: Let me know if it works

File: Dockerfile (hunk 1 above)
Comment (greenw0lf) [reply Dockerfile]: Could you add it maybe to the PR you opened for the example worker?

File: docker-entrypoint.sh
Comment (greenw0lf) [reply docker-entrypoint.sh:1]: @Veldhoen could you test again and see if it works?

**Pull request merged**